agent/rpc_from.go

```diff
@@ -17,11 +17,25 @@ func (a *orbAgent) handleGroupMembership(rpc fleet.GroupMembershipRPCPayload) {
 	// if this is the full list, reset all group subscriptions and subscribed to this list
 	if rpc.FullList {
 		a.unsubscribeGroupChannels()
+		policies, err := a.policyManager.GetRepo().GetAll()
```

Comment:
Unfortunately I think this is too destructive - if I understand correctly, it will remove all running policies every time there is a group change. The problem with this is that the time series will now have a gap in the data, as it throws out the data it had collected for the last minute, and it will take another minute to get it back. What we should really be doing is getting the new list of policies, and then compare that list to the list that is currently running - then add new policies and remove only those no longer in our group. If you think that's too hard then let's discuss.

Reply:
I understand. Where can I get the list of new policies? Every time It receives policies to handle on `handleAgentPolicies`??